clients/fides-js/__tests__/lib/cookie.test.ts

```diff
@@ -44,8 +44,16 @@ mockUuid.v4.mockReturnValue(MOCK_UUID);
 const mockGetCookie = jest.fn((): string | undefined => "mockGetCookie return");
 const mockSetCookie = jest.fn(
   /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
-  (name: string, value: string, attributes: object, encoding: object) =>
-    `mock setCookie return (value=${value})`
+  (name: string, value: string, attributes: object, encoding: object) => {
+    if (
+      ["com", "ca", "org", "uk", "co.uk"].indexOf(
+        (attributes as { domain: string }).domain
+      ) > -1
+    ) {
+      return undefined;
+    }
```

Comment:
Aha, that's pretty reasonable and builds some decent confidence 👍!

In that case, my classic nit is just to provide some breadcrumbs here:
```suggestion
    // Simulate that browsers will not write cookies to known top-level public domains like "com" or "co.uk"
    if (
      ["com", "ca", "org", "uk", "co.uk"].indexOf(
        (attributes as { domain: string }).domain
      ) > -1
    ) {
      return undefined;
    }
```